export function handleException(status) {
    console.log(status);
}
